lib/bigshift/redshift_table_schema.rb

```diff
@@ -8,7 +8,17 @@ def initialize(schema_name, table_name, redshift_connection)
 
     def columns
       @columns ||= begin
-        rows = @redshift_connection.exec_params(%|SELECT "column", "type", "notnull" FROM "pg_table_def" WHERE "schemaname" = $1 AND "tablename" = $2|, [@schema_name, @table_name])
+        query = p %{
+          SELECT "column", "type", "notnull"
+          FROM "pg_table_def" ptd, information_schema.columns isc
+          WHERE ptd.schemaname = isc.table_schema
+          AND ptd.tablename = isc.table_name
+          AND ptd.column = isc.column_name
+          AND "schemaname" = $1
+          AND "tablename" = $2
+          ORDER BY ordinal_position
```

Comment:
I think either all table and column names should be quoted, or none should be. The pieces you've added are not quoted. I'm fine with either and since this SQL is not generated and the names are not mixed case, there is no need for quotes really. I would like it to be consistent, though, so do either or, but not both.

Reply:
In the end I have gone for unquoted with the exception of the columns in the select as `column` is a keyword.

Reply:
👌